src/sage/rings/finite_rings/conway_polynomials.py

```diff
@@ -91,7 +94,10 @@ def exists_conway_polynomial(p, n):
         sage: exists_conway_polynomial(6,6)
         False
     """
-    return sage.databases.conway.ConwayPolynomials().has_polynomial(p,n)
+    try:
+        return ConwayPolynomials().has_polynomial(p,n)
+    except ImportError:
+        return False
```

Comment:
Isn't it incorrect to return `False` when the database is not available? Why catch `ImportError`?

Reply:
The method is named `exists_conway_polynomial`, but it actually only tests whether it is known to Sage. When the database is not installed, nothing is known to Sage. 
Return False here triggers the use of pseudo-Conway polynomials in the relevant code paths.

Reply:
I see. That is reasonable. Thanks.